### PR TITLE
Dathomir updates

### PR DESCRIPTION
**Pull request by Scorchys12**
Removed the red from the areas and updated, added two RP areas and one new dungeon.
One more recipe added and a new creature.

### CODE REVIEW

File: SWLOR.Game.Server/Feature/RecipeDefinition/CookingRecipeDefinition/CookingRecipes.cs [hunk 1]
```diff
@@ -1303,6 +1303,18 @@ private void Tier4()
                 .Component("sanddemon_meat", 3)
                 .Component("r_flour", 2);
 
+            _builder.Create(RecipeType.DathomirPie, SkillType.Agriculture)
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/RecipeDefinition/CookingRecipeDefinition/CookingRecipes.cs:1306]: Can you add this to the bible and send that over to me?

File: SWLOR.Game.Server/Feature/SpawnDefinition/DathomirSpawnDefinition.cs [hunk 2]
```diff
@@ -23,6 +23,7 @@ public class DathomirSpawnDefinition: ISpawnListDefinition
             TarnishedJungles();
             TarnishedJunglesNorth();
             TribeVillage();
+            HidenTunnel();
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/SpawnDefinition/DathomirSpawnDefinition.cs:26]: Typo here. Hiden -> Hidden

File: SWLOR.Game.Server/Feature/RecipeDefinition/CookingRecipeDefinition/CookingRecipes.cs (hunk 1 above)
Comment (Scorchys12) [reply SWLOR.Game.Server/Feature/RecipeDefinition/CookingRecipeDefinition/CookingRecipes.cs:1306]: https://docs.google.com/spreadsheets/d/1KBY3SuVAfFoh3Rt9aH9JPRY1ldzvDmhQPmzXHeHIhm8/edit#gid=1610037751
Dathomir Pie added
Dantooine Flapjack  items updated to reflect the recipe
Krafters Kebab items updated too

File: SWLOR.Game.Server/Feature/LootTableDefinition/DathomirLootTableDefinition.cs [hunk 3]
```diff
@@ -495,6 +496,32 @@ private void JungleBug()
...
+
+        private void GapingSpider()
+        {
+            _builder.Create("DATHOMIR_GAPING_SPIDER")
+                .AddItem("lth_high", 10)
+                .AddItem("lth_imperfect", 5)
+                .AddItem("spider_guts", 10)
+                .AddItem("spider_leg", 5);
+
+            _builder.Create("DATHOMIR_GAPING_SPIDER")
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/LootTableDefinition/DathomirLootTableDefinition.cs:511]: This second one is replacing your first loot table. I think you meant for this to be the rares, but the ID is the same. Please fix.

File: Module/utc/vgapingspider.utc.json [hunk 4]
```diff
@@ -0,0 +1,549 @@
...
+          "type": "cexostring",
+          "value": "LOOT_TABLE_2"
+        },
+        "Type": {
+          "type": "dword",
+          "value": 3
+        },
+        "Value": {
+          "type": "cexostring",
+          "value": "DATHOMIR_GAPING_SPIDER,40,1"
```
Comment (zunath) [inline Module/utc/vgapingspider.utc.json:528]: After you fix the loot table ID mentioned, be sure to update the variable here so it pulls from the correct table.

Comment (Scorchys12) [reply Module/utc/vgapingspider.utc.json]: Fixed

**Comment by zunath** [comment]
Closing because you forgot to include module.ifo.json. I've added it back with the additions to the new areas but can't update your PR. Will submit separately.

**Pull request closed without merging**